doc/runtime.md

```diff
@@ -31,179 +31,17 @@ Optional services:
 
 ### Using CTL's `runtime` overlay
 
-CTL's `overlays.runtime` (contained in its flake `outputs`) provides some mechanisms for conveniently launching all runtime services using [Arion](https://docs.hercules-ci.com/arion) (itself a wrapper around `docker-compose`). To use this, you must have a setup based on Nix flakes (recommended as well for [using CTL as a dependency for Purescript projects](./ctl-as-dependency.md)).
-
-Here is an example that uses the `runtime` overlay to launch all of the required services:
-
-```nix
-{
-
-  inputs = {
-    # You should probably pin this to a specific revision, especially if using
-    # it for Purescript projects
-    cardano-transaction-lib.url = "github:Plutonomicon/cardano-transaction-lib";
-
-    # To use the same version of `nixpkgs` as we do
-    nixpkgs.follows = "cardano-transaction-lib/nixpkgs";
-  };
-
-  outputs = { self, cardano-transaction-lib, nixpkgs, ... }:
-    # some boilerplate
-    let
-      defaultSystems = [
-        "x86_64-linux"
-        "x86_64-darwin"
-        "aarch64-linux"
-        "aarch64-darwin"
-      ];
-      perSystem = nixpkgs.lib.genAttrs defaultSystems;
-
-      # generate `pkgs` with CTL's overlays applied. This gives you access to
-      # various additional packages, using the same versions of CTL
-      nixpkgsFor = system: import nixpkgs {
-        inherit system;
-        overlays = [
-          cardano-transaction-lib.overlays.runtime
-          # This one is optional. If you set `ctl-server.enable = true;`
-          # in the runtime config as shown below, you must enable this
-          # overlay. `ctl-server` itself is **only** required when using
-          # CTL's `applyArgs` effect
-          cardano-transaction-lib.overlays.ctl-server
-          # you probably want this one too, although it's not demonstrated
-          # in this example
-          cardano-transaction-lib.overlays.purescript
-        ];
-      };
-
-      # The configuration for the CTL runtime, which will be passed to the
```

Comment:
all nix code below this point is basically in `nix/runtime.nix` if I got this right. so I migrated the comments to there (and not to `template/ctl-scaffold/flake.nix` as suggested in the issue for this PR).